### PR TITLE
plaso: init at 20230311

### PR DESCRIPTION
**Pull request by JayRovacsek**
###### Description of changes

This pull request includes a range of dependencies as new python package additions to support plaso deployment at most recent versions.
Further to these packages, a minor fix has been applied against pyxattr so that it may be built on darwin hosts successfully

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [x] aarch64-linux
  - [X] x86_64-darwin
  - [X] aarch64-darwin
- [X] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [X] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

Please note this request supersedes https://github.com/NixOS/nixpkgs/pull/224579

### CODE REVIEW

**Comment by JayRovacsek** [comment]
Result of `nixpkgs-review pr 232046` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>189 packages built:</summary>
  <ul>
    <li>bup</li>
    <li>datalad</li>
    <li>datalad.dist</li>
    <li>diffoscope</li>
    <li>diffoscope.dist</li>
    <li>diffoscope.man</li>
    <li>diffoscopeMinimal</li>
    <li>diffoscopeMinimal.dist</li>
    <li>diffoscopeMinimal.man</li>
    <li>git-annex</li>
    <li>git-annex-metadata-gui</li>
    <li>git-annex-metadata-gui.dist</li>
    <li>git-annex.doc</li>
    <li>glusterfs</li>
    <li>python310Packages.acstore</li>
    <li>python310Packages.acstore.dist</li>
    <li>python310Packages.artifacts</li>
    <li>python310Packages.artifacts.dist</li>
    <li>python310Packages.dfdatetime</li>
    <li>python310Packages.dfdatetime.dist</li>
    <li>python310Packages.dfvfs</li>
    <li>python310Packages.dfvfs.dist</li>
    <li>python310Packages.dfwinreg</li>
    <li>python310Packages.dfwinreg.dist</li>
    <li>python310Packages.dtfabric</li>
    <li>python310Packages.dtfabric.dist</li>
    <li>python310Packages.flor</li>
    <li>python310Packages.flor.dist</li>
    <li>python310Packages.git-annex-adapter</li>
    <li>python310Packages.git-annex-adapter.dist</li>
    <li>python310Packages.libbde-python</li>
    <li>python310Packages.libbde-python.dist</li>
    <li>python310Packages.libcreg-python</li>
    <li>python310Packages.libcreg-python.dist</li>
    <li>python310Packages.libesedb-python</li>
    <li>python310Packages.libesedb-python.dist</li>
    <li>python310Packages.libevt-python</li>
    <li>python310Packages.libevt-python.dist</li>
    <li>python310Packages.libevtx-python</li>
    <li>python310Packages.libevtx-python.dist</li>
    <li>python310Packages.libewf-python</li>
    <li>python310Packages.libewf-python.dist</li>
    <li>python310Packages.libfsapfs-python</li>
    <li>python310Packages.libfsapfs-python.dist</li>
    <li>python310Packages.libfsext-python</li>
    <li>python310Packages.libfsext-python.dist</li>
    <li>python310Packages.libfsfat-python</li>
    <li>python310Packages.libfsfat-python.dist</li>
    <li>python310Packages.libfshfs-python</li>
    <li>python310Packages.libfshfs-python.dist</li>
    <li>python310Packages.libfsntfs-python</li>
    <li>python310Packages.libfsntfs-python.dist</li>
    <li>python310Packages.libfsxfs-python</li>
    <li>python310Packages.libfsxfs-python.dist</li>
    <li>python310Packages.libfvde-python</li>
    <li>python310Packages.libfvde-python.dist</li>
    <li>python310Packages.libfwnt-python</li>
    <li>python310Packages.libfwnt-python.dist</li>
    <li>python310Packages.libfwsi-python</li>
    <li>python310Packages.libfwsi-python.dist</li>
    <li>python310Packages.liblnk-python</li>
    <li>python310Packages.liblnk-python.dist</li>
    <li>python310Packages.libluksde-python</li>
    <li>python310Packages.libluksde-python.dist</li>
    <li>python310Packages.libmodi-python</li>
    <li>python310Packages.libmodi-python.dist</li>
    <li>python310Packages.libmsiecf-python</li>
    <li>python310Packages.libmsiecf-python.dist</li>
    <li>python310Packages.libolecf-python</li>
    <li>python310Packages.libolecf-python.dist</li>
    <li>python310Packages.libphdi-python</li>
    <li>python310Packages.libphdi-python.dist</li>
    <li>python310Packages.libqcow-python</li>
    <li>python310Packages.libqcow-python.dist</li>
    <li>python310Packages.libregf-python</li>
    <li>python310Packages.libregf-python.dist</li>
    <li>python310Packages.libscca-python</li>
    <li>python310Packages.libscca-python.dist</li>
    <li>python310Packages.libsigscan-python</li>
    <li>python310Packages.libsigscan-python.dist</li>
    <li>python310Packages.libsmdev-python</li>
    <li>python310Packages.libsmdev-python.dist</li>
    <li>python310Packages.libsmraw-python</li>
    <li>python310Packages.libsmraw-python.dist</li>
    <li>python310Packages.libvhdi-python</li>
    <li>python310Packages.libvhdi-python.dist</li>
    <li>python310Packages.libvmdk-python</li>
    <li>python310Packages.libvmdk-python.dist</li>
    <li>python310Packages.libvsgpt-python</li>
    <li>python310Packages.libvsgpt-python.dist</li>
    <li>python310Packages.libvshadow-python</li>
    <li>python310Packages.libvshadow-python.dist</li>
    <li>python310Packages.libvslvm-python</li>
    <li>python310Packages.libvslvm-python.dist</li>
    <li>python310Packages.plaso</li>
    <li>python310Packages.plaso.dist</li>
    <li>python310Packages.pytsk3</li>
    <li>python310Packages.pytsk3.dist</li>
    <li>python310Packages.pyxattr</li>
    <li>python310Packages.pyxattr.dist</li>
    <li>python311Packages.acstore</li>
    <li>python311Packages.acstore.dist</li>
    <li>python311Packages.artifacts</li>
    <li>python311Packages.artifacts.dist</li>
    <li>python311Packages.dfdatetime</li>
    <li>python311Packages.dfdatetime.dist</li>
    <li>python311Packages.dfvfs</li>
    <li>python311Packages.dfvfs.dist</li>
    <li>python311Packages.dfwinreg</li>
    <li>python311Packages.dfwinreg.dist</li>
    <li>python311Packages.dtfabric</li>
    <li>python311Packages.dtfabric.dist</li>
    <li>python311Packages.flor</li>
    <li>python311Packages.flor.dist</li>
    <li>python311Packages.git-annex-adapter</li>
    <li>python311Packages.git-annex-adapter.dist</li>
    <li>python311Packages.libbde-python</li>
    <li>python311Packages.libbde-python.dist</li>
    <li>python311Packages.libcreg-python</li>
    <li>python311Packages.libcreg-python.dist</li>
    <li>python311Packages.libesedb-python</li>
    <li>python311Packages.libesedb-python.dist</li>
    <li>python311Packages.libevt-python</li>
    <li>python311Packages.libevt-python.dist</li>
    <li>python311Packages.libevtx-python</li>
    <li>python311Packages.libevtx-python.dist</li>
    <li>python311Packages.libewf-python</li>
    <li>python311Packages.libewf-python.dist</li>
    <li>python311Packages.libfsapfs-python</li>
    <li>python311Packages.libfsapfs-python.dist</li>
    <li>python311Packages.libfsext-python</li>
    <li>python311Packages.libfsext-python.dist</li>
    <li>python311Packages.libfsfat-python</li>
    <li>python311Packages.libfsfat-python.dist</li>
    <li>python311Packages.libfshfs-python</li>
    <li>python311Packages.libfshfs-python.dist</li>
    <li>python311Packages.libfsntfs-python</li>
    <li>python311Packages.libfsntfs-python.dist</li>
    <li>python311Packages.libfsxfs-python</li>
    <li>python311Packages.libfsxfs-python.dist</li>
    <li>python311Packages.libfvde-python</li>
    <li>python311Packages.libfvde-python.dist</li>
    <li>python311Packages.libfwnt-python</li>
    <li>python311Packages.libfwnt-python.dist</li>
    <li>python311Packages.libfwsi-python</li>
    <li>python311Packages.libfwsi-python.dist</li>
    <li>python311Packages.liblnk-python</li>
    <li>python311Packages.liblnk-python.dist</li>
    <li>python311Packages.libluksde-python</li>
    <li>python311Packages.libluksde-python.dist</li>
    <li>python311Packages.libmodi-python</li>
    <li>python311Packages.libmodi-python.dist</li>
    <li>python311Packages.libmsiecf-python</li>
    <li>python311Packages.libmsiecf-python.dist</li>
    <li>python311Packages.libolecf-python</li>
    <li>python311Packages.libolecf-python.dist</li>
    <li>python311Packages.libphdi-python</li>
    <li>python311Packages.libphdi-python.dist</li>
    <li>python311Packages.libqcow-python</li>
    <li>python311Packages.libqcow-python.dist</li>
    <li>python311Packages.libregf-python</li>
    <li>python311Packages.libregf-python.dist</li>
    <li>python311Packages.libscca-python</li>
    <li>python311Packages.libscca-python.dist</li>
    <li>python311Packages.libsigscan-python</li>
    <li>python311Packages.libsigscan-python.dist</li>
    <li>python311Packages.libsmdev-python</li>
    <li>python311Packages.libsmdev-python.dist</li>
    <li>python311Packages.libsmraw-python</li>
    <li>python311Packages.libsmraw-python.dist</li>
    <li>python311Packages.libvhdi-python</li>
    <li>python311Packages.libvhdi-python.dist</li>
    <li>python311Packages.libvmdk-python</li>
    <li>python311Packages.libvmdk-python.dist</li>
    <li>python311Packages.libvsgpt-python</li>
    <li>python311Packages.libvsgpt-python.dist</li>
    <li>python311Packages.libvshadow-python</li>
    <li>python311Packages.libvshadow-python.dist</li>
    <li>python311Packages.libvslvm-python</li>
    <li>python311Packages.libvslvm-python.dist</li>
    <li>python311Packages.plaso</li>
    <li>python311Packages.plaso.dist</li>
    <li>python311Packages.pytsk3</li>
    <li>python311Packages.pytsk3.dist</li>
    <li>python311Packages.pyxattr</li>
    <li>python311Packages.pyxattr.dist</li>
    <li>qemu_full</li>
    <li>qemu_full.debug</li>
    <li>qemu_full.ga</li>
  </ul>
</details>

**Comment by JayRovacsek** [comment]
Result of `nixpkgs-review pr 232046` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>python311Packages.plaso</li>
    <li>python311Packages.plaso.dist</li>
  </ul>
</details>
<details>
  <summary>166 packages built:</summary>
  <ul>
    <li>python310Packages.acstore</li>
    <li>python310Packages.acstore.dist</li>
    <li>python310Packages.artifacts</li>
    <li>python310Packages.artifacts.dist</li>
    <li>python310Packages.dfdatetime</li>
    <li>python310Packages.dfdatetime.dist</li>
    <li>python310Packages.dfvfs</li>
    <li>python310Packages.dfvfs.dist</li>
    <li>python310Packages.dfwinreg</li>
    <li>python310Packages.dfwinreg.dist</li>
    <li>python310Packages.dtfabric</li>
    <li>python310Packages.dtfabric.dist</li>
    <li>python310Packages.flor</li>
    <li>python310Packages.flor.dist</li>
    <li>python310Packages.libbde-python</li>
    <li>python310Packages.libbde-python.dist</li>
    <li>python310Packages.libcreg-python</li>
    <li>python310Packages.libcreg-python.dist</li>
    <li>python310Packages.libesedb-python</li>
    <li>python310Packages.libesedb-python.dist</li>
    <li>python310Packages.libevt-python</li>
    <li>python310Packages.libevt-python.dist</li>
    <li>python310Packages.libevtx-python</li>
    <li>python310Packages.libevtx-python.dist</li>
    <li>python310Packages.libewf-python</li>
    <li>python310Packages.libewf-python.dist</li>
    <li>python310Packages.libfsapfs-python</li>
    <li>python310Packages.libfsapfs-python.dist</li>
    <li>python310Packages.libfsext-python</li>
    <li>python310Packages.libfsext-python.dist</li>
    <li>python310Packages.libfsfat-python</li>
    <li>python310Packages.libfsfat-python.dist</li>
    <li>python310Packages.libfshfs-python</li>
    <li>python310Packages.libfshfs-python.dist</li>
    <li>python310Packages.libfsntfs-python</li>
    <li>python310Packages.libfsntfs-python.dist</li>
    <li>python310Packages.libfsxfs-python</li>
    <li>python310Packages.libfsxfs-python.dist</li>
    <li>python310Packages.libfvde-python</li>
    <li>python310Packages.libfvde-python.dist</li>
    <li>python310Packages.libfwnt-python</li>
    <li>python310Packages.libfwnt-python.dist</li>
    <li>python310Packages.libfwsi-python</li>
    <li>python310Packages.libfwsi-python.dist</li>
    <li>python310Packages.liblnk-python</li>
    <li>python310Packages.liblnk-python.dist</li>
    <li>python310Packages.libluksde-python</li>
    <li>python310Packages.libluksde-python.dist</li>
    <li>python310Packages.libmodi-python</li>
    <li>python310Packages.libmodi-python.dist</li>
    <li>python310Packages.libmsiecf-python</li>
    <li>python310Packages.libmsiecf-python.dist</li>
    <li>python310Packages.libolecf-python</li>
    <li>python310Packages.libolecf-python.dist</li>
    <li>python310Packages.libphdi-python</li>
    <li>python310Packages.libphdi-python.dist</li>
    <li>python310Packages.libqcow-python</li>
    <li>python310Packages.libqcow-python.dist</li>
    <li>python310Packages.libregf-python</li>
    <li>python310Packages.libregf-python.dist</li>
    <li>python310Packages.libscca-python</li>
    <li>python310Packages.libscca-python.dist</li>
    <li>python310Packages.libsigscan-python</li>
    <li>python310Packages.libsigscan-python.dist</li>
    <li>python310Packages.libsmdev-python</li>
    <li>python310Packages.libsmdev-python.dist</li>
    <li>python310Packages.libsmraw-python</li>
    <li>python310Packages.libsmraw-python.dist</li>
    <li>python310Packages.libvhdi-python</li>
    <li>python310Packages.libvhdi-python.dist</li>
    <li>python310Packages.libvmdk-python</li>
    <li>python310Packages.libvmdk-python.dist</li>
    <li>python310Packages.libvsgpt-python</li>
    <li>python310Packages.libvsgpt-python.dist</li>
    <li>python310Packages.libvshadow-python</li>
    <li>python310Packages.libvshadow-python.dist</li>
    <li>python310Packages.libvslvm-python</li>
    <li>python310Packages.libvslvm-python.dist</li>
    <li>python310Packages.plaso</li>
    <li>python310Packages.plaso.dist</li>
    <li>python310Packages.pytsk3</li>
    <li>python310Packages.pytsk3.dist</li>
    <li>python310Packages.pyxattr</li>
    <li>python310Packages.pyxattr.dist</li>
    <li>python311Packages.acstore</li>
    <li>python311Packages.acstore.dist</li>
    <li>python311Packages.artifacts</li>
    <li>python311Packages.artifacts.dist</li>
    <li>python311Packages.dfdatetime</li>
    <li>python311Packages.dfdatetime.dist</li>
    <li>python311Packages.dfvfs</li>
    <li>python311Packages.dfvfs.dist</li>
    <li>python311Packages.dfwinreg</li>
    <li>python311Packages.dfwinreg.dist</li>
    <li>python311Packages.dtfabric</li>
    <li>python311Packages.dtfabric.dist</li>
    <li>python311Packages.flor</li>
    <li>python311Packages.flor.dist</li>
    <li>python311Packages.libbde-python</li>
    <li>python311Packages.libbde-python.dist</li>
    <li>python311Packages.libcreg-python</li>
    <li>python311Packages.libcreg-python.dist</li>
    <li>python311Packages.libesedb-python</li>
    <li>python311Packages.libesedb-python.dist</li>
    <li>python311Packages.libevt-python</li>
    <li>python311Packages.libevt-python.dist</li>
    <li>python311Packages.libevtx-python</li>
    <li>python311Packages.libevtx-python.dist</li>
    <li>python311Packages.libewf-python</li>
    <li>python311Packages.libewf-python.dist</li>
    <li>python311Packages.libfsapfs-python</li>
    <li>python311Packages.libfsapfs-python.dist</li>
    <li>python311Packages.libfsext-python</li>
    <li>python311Packages.libfsext-python.dist</li>
    <li>python311Packages.libfsfat-python</li>
    <li>python311Packages.libfsfat-python.dist</li>
    <li>python311Packages.libfshfs-python</li>
    <li>python311Packages.libfshfs-python.dist</li>
    <li>python311Packages.libfsntfs-python</li>
    <li>python311Packages.libfsntfs-python.dist</li>
    <li>python311Packages.libfsxfs-python</li>
    <li>python311Packages.libfsxfs-python.dist</li>
    <li>python311Packages.libfvde-python</li>
    <li>python311Packages.libfvde-python.dist</li>
    <li>python311Packages.libfwnt-python</li>
    <li>python311Packages.libfwnt-python.dist</li>
    <li>python311Packages.libfwsi-python</li>
    <li>python311Packages.libfwsi-python.dist</li>
    <li>python311Packages.liblnk-python</li>
    <li>python311Packages.liblnk-python.dist</li>
    <li>python311Packages.libluksde-python</li>
    <li>python311Packages.libluksde-python.dist</li>
    <li>python311Packages.libmodi-python</li>
    <li>python311Packages.libmodi-python.dist</li>
    <li>python311Packages.libmsiecf-python</li>
    <li>python311Packages.libmsiecf-python.dist</li>
    <li>python311Packages.libolecf-python</li>
    <li>python311Packages.libolecf-python.dist</li>
    <li>python311Packages.libphdi-python</li>
    <li>python311Packages.libphdi-python.dist</li>
    <li>python311Packages.libqcow-python</li>
    <li>python311Packages.libqcow-python.dist</li>
    <li>python311Packages.libregf-python</li>
    <li>python311Packages.libregf-python.dist</li>
    <li>python311Packages.libscca-python</li>
    <li>python311Packages.libscca-python.dist</li>
    <li>python311Packages.libsigscan-python</li>
    <li>python311Packages.libsigscan-python.dist</li>
    <li>python311Packages.libsmdev-python</li>
    <li>python311Packages.libsmdev-python.dist</li>
    <li>python311Packages.libsmraw-python</li>
    <li>python311Packages.libsmraw-python.dist</li>
    <li>python311Packages.libvhdi-python</li>
    <li>python311Packages.libvhdi-python.dist</li>
    <li>python311Packages.libvmdk-python</li>
    <li>python311Packages.libvmdk-python.dist</li>
    <li>python311Packages.libvsgpt-python</li>
    <li>python311Packages.libvsgpt-python.dist</li>
    <li>python311Packages.libvshadow-python</li>
    <li>python311Packages.libvshadow-python.dist</li>
    <li>python311Packages.libvslvm-python</li>
    <li>python311Packages.libvslvm-python.dist</li>
    <li>python311Packages.pytsk3</li>
    <li>python311Packages.pytsk3.dist</li>
    <li>python311Packages.pyxattr</li>
    <li>python311Packages.pyxattr.dist</li>
  </ul>
</details>

**Comment by JayRovacsek** [comment]
@AndersonTorres apologies for the @, any recommendations on what could be changed to make this PR visible to someone who can ✅ ?
Simply reaching out as you were able to give feedback previously: https://github.com/NixOS/nixpkgs/pull/224579#issuecomment-1544321881

File: pkgs/development/python-modules/acstore/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{ buildPythonPackage, fetchPypi, lib }:
+
+buildPythonPackage rec {
+  pname = "acstore";
+
+  version = "20230226";
```
Comment (fabaff) [inline pkgs/development/python-modules/acstore/default.nix:6]: The latest release would be `20240407`.

```suggestion
  pname = "acstore";
  version = "20230226";
  pyproject = true;
```

and add `build-system = [ setuptools ];` after `src`.

File: pkgs/development/python-modules/acstore/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
```
Comment (fabaff) [inline pkgs/development/python-modules/acstore/default.nix:7]: Later releases will have a constraint on [Python > 3.8](https://github.com/log2timeline/acstore/blob/20240407/setup.cfg#L25).

```suggestion

  disabled = pythonOlder "3.8";
```

File: pkgs/development/python-modules/acstore/default.nix [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-MDeWOU1cv6dW8LL2UmdUbSwm0vg4YfxvpTDTb15+/+s=";
```
Comment (fabaff) [inline pkgs/development/python-modules/acstore/default.nix:10]: It's a SRI hash, thus `hash`.

```suggestion
    hash = "sha256-MDeWOU1cv6dW8LL2UmdUbSwm0vg4YfxvpTDTb15+/+s=";
```

File: pkgs/development/python-modules/acstore/default.nix [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  };
+
```
Comment (fabaff) [inline pkgs/development/python-modules/acstore/default.nix:12]: The upstream repo contains [tests](https://github.com/log2timeline/acstore/tree/main/tests). We should run them if possible,

```suggestion

  pythonImportsCheck = [
    "acstore"
  ];
```

File: pkgs/development/python-modules/acstore/default.nix [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  meta = with lib; {
+    description = "ACStore, or Attribute Container Storage, provides a stand-alone implementation to read and write attribute container storage files";
+    downloadPage = "https://github.com/log2timeline/acstore/releases";
+    homepage = "https://github.com/log2timeline/acstore";
```
Comment (fabaff) [inline pkgs/development/python-modules/acstore/default.nix:16]: Not much content in the changelog at the moment, but perhaps there will be more in the future.

```suggestion
    homepage = "https://github.com/log2timeline/acstore";
    changelog = "https://github.com/log2timeline/acstore/releases/tag/${version}";
```

File: pkgs/development/python-modules/acstore/default.nix [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    license = licenses.asl20;
+    maintainers = [ maintainers.jayrovacsek ];
+    platforms = platforms.unix;
```
Comment (fabaff) [inline pkgs/development/python-modules/acstore/default.nix:19]: I can't find a platform limitation in the upstream repo.

```suggestion
```

File: pkgs/development/python-modules/artifacts/default.nix [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  pname = "artifacts";
+
+  version = "20221219";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-rCmmzA9OmUiJPVrOJu2N61zj6Z4MysJP6fOTKRibCm0=";
+  };
+
+  propagatedBuildInputs = [ pyyaml ];
```
Comment (fabaff) [inline pkgs/development/python-modules/artifacts/default.nix:13]: Was changed recently.

```suggestion
  dependencies = [ pyyaml ];
```

File: pkgs/development/python-modules/dfvfs/default.nix [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+    libvmdk-python
+    libvsgpt-python
+    libvshadow-python
+    libvslvm-python
+    pytsk3
+    pyxattr
+    pyyaml
+  ];
+
+  doCheck = false;
```
Comment (fabaff) [inline pkgs/development/python-modules/dfvfs/default.nix:77]: A comment would be helpful why we can't run the tests.

**Review by fabaff — COMMENTED** [review]
My comments may apply for the other modules as well.

**Comment by JayRovacsek** [comment]
@fabaff thanks so much for your review, I'll apply changes as suggested and resolve missing tests (actually have completed this work on my local instance of these as per [this commit](https://github.com/JayRovacsek/nix-config/commit/3cd203e37239409dd4ebf1be246a90fcd5139c90) + have applied latest updates as of time of commit :grin:)

Given the age of this PR, need to rebase to resolve conflicts and large number of changes are you okay with me tagging you in on a new PR I create with all of the issues you've raised resolved?

**Pull request closed without merging**